app/views/datafiles/files.html.erb

```diff
@@ -0,0 +1,39 @@
+
+<% content_for :page_title do %>
+    Links to your data -
+<% end %>
+
+<main role="main" id="content">
+  <h1 class="heading-large">Links to your data</h1>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Link name</th>
+        <th>Valid</th>
+        <th><span class="visuallyhidden">Actions</span></th>
+      </tr>
+    </thead>
+  <tbody>
+    <% @datafiles.each do |link| %>
+      <tr id="file_1">
+        <th><a href="<%= link.url %>"><%= link.name %></a></th>
+        <td>✔</td>
+        <td class="actions">
+          <%= link_to 'Edit', edit_file_path(@dataset.id, link.id) %>
+          <%= link_to 'Delete', file_path(@dataset.id, link.id), method: :delete %>
```

Comment:
I think you can just pass in `@dataset`, I thought it was bad juju to do `@dataset.id`